src/components/FilterTodo.tsx

```diff
@@ -0,0 +1,46 @@
+import { useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faList,
+  faSpinner,
+  faSquareCheck,
+  faTrash
+} from '@fortawesome/free-solid-svg-icons';
+
+import Button from './Button';
+import { filters, todosActions } from '../store/store';
+import classes from './FilterTodo.module.css';
+
+function FilterTodo() {
+  const dispatch = useDispatch();
+
+  const { filterBy, removeCompleted } = todosActions;
+
+  const showAllHandler = () => dispatch(filterBy(filters.ALL));
+  const showActiveHandler = () => dispatch(filterBy(filters.ACTIVE));
+  const showCompletedHandler = () => dispatch(filterBy(filters.COMPLETED));

```

Comment:
Can't it be one function and pass param ? If you have 7 filters, will you create 7 functions ?